build/root/Makefile

```diff
@@ -56,10 +56,6 @@ $(error Both KUBE_GOFLAGS and GOFLAGS are set. Please use just GOFLAGS)
 endif
 endif
 
-# Extra options for the release or quick-release options:
```

Comment:
Why do we need to remove this?

Reply:
unless my local testing is wrong, line 60,61 are useless because they been set to default value at line 455,456

Reply:
@figo then can we please remove it in another PR, not this one. (unless this stops you from doing what you need for setting up another registry)

Reply:
make sense, will remove it from this PR